docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -52,7 +52,7 @@ Run the following commands:
 ```bash
 export latest_kube=<latest kubernetes version e.g. v1.18.0>
 sudo sed -i "s|$(grep KUBELET_IMAGE_URL /etc/kubernetes/kubelet.env)|KUBELET_IMAGE_URL=quay.io/poseidon/kubelet|g" /etc/kubernetes/kubelet.env
-sudo sed -i "s|$(grep -i kubelet_image_tag /etc/kubernetes/kubelet.env)|KUBELET_IMAGE_TAG=${latest_kube}|g" /etc/kubernetes/kubelet.env
+sudo sed -i "s|$(grep KUBELET_IMAGE_TAG /etc/kubernetes/kubelet.env)|KUBELET_IMAGE_TAG=${latest_kube}|g" /etc/kubernetes/kubelet.env
```

Comment:
Why not something like `s/^KUBELET_IMAGE_TAG=.*/KUBELET_IMAGE_TAG=${latest_kube}/g` ? Or even `.*KUBELET_IMAGE_TAG.*` as this is what `grep` is doing.

Reply:
What purpose does `.*KUBELET_IMAGE_TAG.*` serve?

Reply:
There is no need for grep in the subshell. The exact same regex matching sed will do for you.

Reply:
TIL. Thanks @invidian :+1: